ui/raidboss/data/06-ew/dungeon/stigma_dreamscape.txt

```diff
@@ -0,0 +1,125 @@
+### Stigma Dreamscape
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839::.*is no longer sealed/ window 100000 jump 0
+
+
+#~~~~~~~~~~~~~#
+# PROTO-OMEGA #
+#~~~~~~~~~~~~~#
+
+0.0 "--sync--" sync / 00:0839::A-4 Command will be sealed off/ window 0,1
+12.6 "Side Cannons" sync / 1[56]:[^:]*:Proto-Omega:(6320|6321):/ window 12.6,10
+24.2 "Forward Interceptors/Rear Interceptors" sync / 1[56]:[^:]*:Proto-Omega:(6322|6324):/
+29.3 "Chemical Missile" sync / 1[56]:[^:]*:Proto-Omega:6328:/
+34.4 "Burn" sync / 1[56]:[^:]*:Proto-Omega:6329:/
+40.6 "Electric Slide" sync / 1[56]:[^:]*:Proto-Omega:632A:/ window 30,30
+49.7 "Guided Missile" sync / 1[56]:[^:]*:Proto-Omega:6326:/
+
+65.6 "Iron Kiss" sync / 1[56]:[^:]*:Mark II Guided Missile:6327:/
+73.0 "Electric Slide" sync / 1[56]:[^:]*:Proto-Omega:632A:/
+84.2 "Mustard Bomb" sync / 1[56]:[^:]*:Proto-Omega:632B:/
+96.4 "Side Cannons" sync / 1[56]:[^:]*:Proto-Omega:(6320|6321):/ window 10,10
+104.6 "Guided Missile" sync / 1[56]:[^:]*:Proto-Omega:6326:/
+111.7 "Chemical Missile" sync / 1[56]:[^:]*:Proto-Omega:6328:/
+116.8 "Burn" sync / 1[56]:[^:]*:Proto-Omega:6329:/
+
+# TODO This is likely inaccurate past Mustard Bomb.
```

Comment:
Do you think it's worth guessing that this is a loop for now?

Reply:
I suppose there's no reason not to, you're right.

Reply:
Hmm, looking at this more, I'm not completely sure this *is* the loop. I had left it just as it was generated for the initial version, but for the loop I went and adjusted the previous section to be the lookahead, and it's definitely got different timings.

I think this is probably safe to merge for now, but we definitely need more data.